### ISSUE TITLE
update rqcfilter.wdl to be compatible with new shared workflow requirements

### ISSUE THREAD

**Issue by aclum** [microbiomedata/ReadsQC#24]
see https://docs.google.com/document/d/1gqpopBC1ovWR7p5WFilI-Kn_BYOSVBkX4fyDg-8nlX0/edit for requirements.
At a minimum we need to remove the runtime_minutes to be able to run with JAWS, note this is also filed as a bug w/JAWS since this should be ignored.

https://code.jgi.doe.gov/advanced-analysis/jaws-support/-/issues/118

**Comment by aclum** [comment]
We are running on JAWS now but this is still relevant, recommended tasks from the JAWS team were to specify:
cpu
runtime_minutes

**Comment by aclum** [comment]
relates to/duplicat of https://github.com/microbiomedata/ReadsQC/issues/61.
